st2common/st2common/services/coordination.py

```diff
@@ -51,16 +55,30 @@ def heartbeat(self):
         return True
 
 
+class NoOpAsyncResult(object):
```

Comment:
Can you provide a short description why we need to wrap result with this class?

Reply:
It's to correctly mimic tooz API - most of the tooz methods return an async result (future).

I can add that as a docstring.